source/jenkins/upgrade-java.html.md.erb

```diff
@@ -0,0 +1,76 @@
+
+To upgrade SDS SBOX do the following:
+
+1. If you have not already done so create a PR in the [CNP Jenkins repo](https://github.com/hmcts/cnp-jenkins-docker/blob/master/jenkins/Dockerfile) with the Java version incremented. You can find updated Jenkins Base image tag from the [Jenkins dockerhub repo](https://hub.docker.com/r/jenkins/jenkins/tags).
+2. Creating the PR should have automatically created a docker image with a pr- in it's name.
+3. Check the [container registry](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/8999dec3-0104-4a27-94ee-6588559729d1/resourceGroups/rpe-acr-prod-rg/providers/Microsoft.ContainerRegistry/registries/hmctspublic/repository) for the updated tag.
+4. Point the SDS flux config to use that new tag by updating [PTLSBOX](https://github.com/hmcts/sds-flux-config/blob/master/apps/jenkins/jenkins/ptlsbox/jenkins-controller-version.yaml).
+5. Then verify that SDS Jenkins is working as expected by running [Toffee tests](https://sds-sandbox-build.hmcts.net/blue/organizations/jenkins/HMCTS_Nightly_Sandbox/sds-toffee-frontend/).
+
+Do the same for CFT by running the Plum tests.
+If everything is working as expected, you can proceed with the PROD upgrade.
+
+## Upgrade PROD SDS Jenkins
+A change request should not be needed for this, but be vigilant of any issues by checking slack and monitoring the deployment.
+
+Firstly merge in the changes to the Docker image that you made earlier. Make note of the tag that is created when you merge it in by checking the build log.
+This alone won't upgrade the Java version as you will need to update the tag that is used. Upon merge, check the [container registry](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/8999dec3-0104-4a27-94ee-6588559729d1/resourceGroups/rpe-acr-prod-rg/providers/Microsoft.ContainerRegistry/registries/hmctspublic/repository) for the updated tag. You should also be able to see it in the build log when you merge your changes.
+
+1. If you have not already done so create a PR in the [CNP Jenkins repo](https://github.com/hmcts/cnp-jenkins-docker/blob/master/jenkins/Dockerfile) with the Java version incremented. You can find updated Jenkins Base image tag from the [Jenkins dockerhub repo](https://hub.docker.com/r/jenkins/jenkins/tags).
+2. Bump the Docker image tag for [PTLSBOX](https://github.com/hmcts/sds-flux-config/blob/master/apps/jenkins/jenkins/ptlsbox/jenkins-controller-version.yaml)
+3. Bump the Docker image tag for [SDS](https://github.com/hmcts/sds-flux-config/blob/master/apps/jenkins/jenkins/jenkins-controller-version.yaml)
+4. Point the SDS flux config to use that new tag by updating [this]([PROD](https://github.com/hmcts/sds-flux-config/blob/master/apps/jenkins/jenkins/jenkins-controller-version.yaml))
+5. After 10 - 15 minutes verify that the upgrade was successful by running [Toffee tests](https://sds-build.hmcts.net/blue/organizations/jenkins/HMCTS_Nightly_Sandbox/sds-toffee-frontend/).
+
+## Upgrade PROD CFT Jenkins
```

Comment:
I also think we only need one prod section with example links to both places instead of all steps written out twice

Reply:
@DanielKChristou Almost, I still think this can be combined into one `Upgrade production Jenkins Controller` section with some cft/sds links, what do you think?
![image](https://github.com/user-attachments/assets/975c20a6-c1ee-4ca9-921d-e3ce9d8c9c83)

Reply:
Probably for the best as a lot of the steps are similar and I already went in that direction for the Agents section.